isdf/CMakeLists.txt

```diff
@@ -0,0 +1,204 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(isdf)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  rospy
+)
+
+## System dependencies are found with CMake's conventions
```

Comment:
Best to remove all comment code (here and other files) so there is no confusion for users.